spec/menu_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'menu'
+
+describe Menu do
+it 'must show a list of dishes with prices' do
+  menu=Menu.new
+expect(menu.dish_list).to eq ({:chicken => 3, :beef => 4, :pizza => 5})
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)